node/sc/sub_bridge_handler.go

```diff
@@ -122,6 +123,25 @@ func (sbh *SubBridgeHandler) getParentOperatorBalance() (*big.Int, error) {
 	return sbh.subbridge.remoteBackend.BalanceAt(ctx, sbh.subbridge.bridgeAccounts.pAccount.address, nil)
 }
 
+// getParentBridgeContractBalance returns the parent bridge contract's balance.
+func (sbh *SubBridgeHandler) getParentBridgeContractBalance(addr common.Address) (*big.Int, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	found := false
+	for _, bridgeContractAddr := range sbh.subbridge.bridgeManager.GetAllParentBridgeAddrs() {
```

Comment:
What do you think?

1. merge this first and later implement alias.
2. develop this code base with alias

Reply:
I'm okay with anything. but the separate PR would be nice in view of reviewers.

Reply:
Okay, then let's go with the first. Thanks.